pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -142,6 +148,8 @@ private static long getRandomInitialDelayInSeconds() {
 
     private static final int DEFAULT_SEGMENT_LEVEL_VALIDATION_INTERVAL_IN_SECONDS = 24 * 60 * 60;
     private static final int DEFAULT_SEGMENT_RELOCATOR_FREQUENCY_IN_SECONDS = 60 * 60;
+
+    private static final int DEFAULT_VALIDATION_RANGE_IN_DAYS_TO_CHECK_MISSING_SEGMENT_STORE_COPY = 3;
```

Comment:
Please keep the units at the end of the member name (*_IN_DAYS).
Also, try to shorten the name if possible